pkg/http/api/v1/handler_test.go

```diff
@@ -418,6 +419,9 @@ func TestRequestHandler_GetReady(t *testing.T) {
 }
 
 func TestRequestHandler_GetMetadata(t *testing.T) {
+	err := os.Setenv("TRIVY_VERSION", "0.1.6")
```

Comment:
This will cause side effects for any following tests that get run after this as it's not cleared/cleaned up.